### PR TITLE
Tooltips: Remove Choice item boost from Dynamaxed Pokemon

### PR DESCRIPTION
**Pull request by KrisXV**
I also added tooltips for the defender-based Utility Umbrella changes

### CODE REVIEW

**Comment by Zarel** [comment]
I don't think this affects the first turn of dynamaxing yet?

File: src/battle-tooltips.ts [hunk 1]
```diff
@@ -1192,7 +1204,7 @@ class BattleTooltips {
 		if (move.id === 'blizzard') {
 			value.weatherModify(0, 'Hail');
 		}
-		if (move.id === 'hurricane' || move.id === 'thunder') {
+		if ((move.id === 'hurricane' || move.id === 'thunder')) {
```
Comment (Zarel) [inline src/battle-tooltips.ts]: ?

Comment (KrisXV) [reply src/battle-tooltips.ts]: hehe oops

**Comment by KrisXV** [comment]
no it doesn't @Zarel because the pokemon doesn't have the volatile applied yet

**Comment by Zarel** [comment]
I mean, it should if you check the "Dynamax" button.

**Comment by KrisXV** [comment]
how do i do that

**Comment by Zarel** [comment]
It's not currently possible, you'd have to pass in another parameter or grab it from somewhere.

**Pull request merged**